examples/ios-objc/ruddertyper.yml

```
@@ -0,0 +1,15 @@
+# RudderStack RudderTyper Configuration 
+# Just run `npx rudder-typer` to re-generate a client with the latest versions of these events.
+
+client:
+  sdk: analytics-ios
+  language: objective-c
+
+trackingPlans:
+  # Tracking Plan: Simple Plan
+  # https://app.rudderstack.com/trackingplans/tp_2keoGz7isp0AwGYznhExEigY1CX
+  - id: tp_2keoGz7isp0AwGYznhExEigY1CX
+    version: 2
+    workspaceSlug: 2h8hLQOPxmXc4kdECTy9fxt3ow4
```

Let's anonymise these values:
```suggestion
  # https://app.rudderstack.com/trackingplans/trackingPlanId
  - id: trackingPlanId
    version: 2
    workspaceSlug: workspaceId
```